airflow/models/taskinstance.py

```diff
@@ -347,18 +347,18 @@ def generate_command(dag_id,
         """
         iso = execution_date.isoformat()
         cmd = ["airflow", "tasks", "run", str(dag_id), str(task_id), str(iso)]
-        cmd.extend(["--mark_success"]) if mark_success else None
+        cmd.extend(["--mark-success"]) if mark_success else None
```

Comment:
huh?

```suggestion
        if mark_success:
            cmd.extend(["--mark-success"])
```

I think the usage of `if else` here is bad.

Reply:
Same below.

Reply:
Good catch!